### PR TITLE
plant species topology for release/102

### PR DESCRIPTION
**Pull request by brunocontrerasmoreira**
## Description

This is the updated Newick file with the topology of plant species as of release 49/102.
It includes only 3 new species. This should serve as well as the list of plant species for this Compara production cycle @JAlvarezJarreta @dthybert 
The new UK wheat cultivars are not included yet as these still have lift-over annotation and we prefer to wait until their de novo annotation is in place @gnaamati

### CODE REVIEW

File: conf/plants/species_tree.topology.nw [hunk 1]
```diff
@@ -244,7 +250,16 @@
           )Anacardiaceae,
          )malvids,
         )rosids,
-        beta_vulgaris,
+        ( 
+         )
```
Comment (JAlvarezJarreta) [inline conf/plants/species_tree.topology.nw:254]: I think this parenthesis is in the wrong direction:
```suggestion
         (
```

Comment (brunocontrerasmoreira) [reply conf/plants/species_tree.topology.nw]: Thanks for catching that one, fixed in the following commit

**Comment by brunocontrerasmoreira** [comment]
It works with iToL, I will add the trailing commas just in case

**Comment by brunocontrerasmoreira** [comment]
You're right this format is weird, it never worked with my favourite FigTree software, but it's the format we have used time and again. It does work with iToL. We can change it if you provide a description of what you want.

**Comment by JAlvarezJarreta** [comment]
Although other formatting may be better to see errors in a visual inspection (for instance, use tabs of 4 spaces instead of 1), I don't think it is necessary to change it now. Provided the tree is correct and with the right amount of species, that's all we need.

**Pull request merged**